AmericanMedicalAssociation/ama-style-guide-2
# ADF-259: Add Sticky JS to Drupal for Find Dr Main Navigation.

**Jira Ticket**
- [ADF-259: Add Sticky JS to Drupal for Find Dr Main Navigation.](https://palantir.atlassian.net/browse/ADF-259)

## Description

Add Sticky JS for the Main Navigation for profile users and Administration role.


## To Test

- Checkout the `ADF-259-Add-dr-finder-sticky-JS` branch and cd into style guide, run composer install

- Copy the code from `dr-finder-sticky.js` file under ` styleguide/source/assets/js`

- Paste the code into the `dr-finder-sticky.js` file under ` styleguide/public/assets/js` in the `dr-finder` Repo.

- Go to `https://dr-finder.ddev.site/search` log in as Admin to test if the Navigation bar response properly with admin toolbar; And log out to see if it response properly.
